### PR TITLE
fix: Move Close Icon from Account to Arrows

### LINKED ISSUE #257: Close Full Icon is in the Account Category instead of Arrows

### PR DESCRIPTION
**Pull request by MarinaRomanova**
## 📋 Changes description
CloseFull icon should be a part of Arrows icon

## ✅ Checklist
- [ ] closes #257

### CODE REVIEW

**Review by SimonMarquis — APPROVED** [review]
LGTM

File: spark-icons/src/main/kotlin/com/adevinta/spark/icons/SparkIcon.kt [hunk 1]
```diff
@@ -144,7 +145,11 @@ public sealed class SparkIcon(@DrawableRes override val drawableId: Int) : IconD
             public object Right : Arrow(R.drawable.spark_icons_chevron_double_right)
         }
 
-        public object Close : Arrows(R.drawable.spark_icons_close)
+        public sealed class Close(@DrawableRes override val drawableId: Int) : Arrows(drawableId) {
+            public object Default : Arrow(R.drawable.spark_icons_close)
```
Comment (soulcramer) [inline spark-icons/src/main/kotlin/com/adevinta/spark/icons/SparkIcon.kt:149]: `Default` & `Full` should extend `Close` instead of `Arrows`

**Review by soulcramer — DISMISSED** [review]

File: spark-icons/src/main/kotlin/com/adevinta/spark/icons/SparkIcon.kt (hunk 1 above)
Comment (SimonMarquis) [reply spark-icons/src/main/kotlin/com/adevinta/spark/icons/SparkIcon.kt:149]: 🙈 good catch!

Comment (MarinaRomanova) [reply spark-icons/src/main/kotlin/com/adevinta/spark/icons/SparkIcon.kt]: Yes, indeed! I have corrected it in the next commit

**Pull request merged**